modules/images-configuration-image-registry-settings-hcp.adoc

```diff
@@ -0,0 +1,143 @@
+
+Name:
+Domain Prefix:
+Display Name:
+ID:
+External ID:
+Control Plane:              ROSA Service Hosted
+OpenShift Version:          4.16.12
+Channel Group:              stable
+DNS:                        Not ready
+AWS Account:                765374464689
+AWS Billing Account:        765374464689
+API URL:
+Console URL:
+Region:                     us-west-2
+Availability:
+ - Control Plane:           MultiAZ
+ - Data Plane:              SingleAZ
+
+Nodes:
+ - Compute (desired):       2
+ - Compute (current):       0
+Network:
+ - Type:                    OVNKubernetes
+ - Service CIDR:            <service_cidr>
+ - Machine CIDR:            <machine_cidr>
+ - Pod CIDR:                <pod_cidr>
+ - Host Prefix:             /23
+ - Subnets:                 <subnet_ids>
+EC2 Metadata Http Tokens:   optional
+Role (STS) ARN:             arn:aws:iam::765374464689:role/magchen-local-HCP-ROSA-Installer-Role
+Support Role ARN:           arn:aws:iam::765374464689:role/magchen-local-HCP-ROSA-Support-Role
+Instance IAM Roles:
+ - Worker:                  arn:aws:iam::765374464689:role/magchen-local-HCP-ROSA-Worker-Role
+Operator IAM Roles:
+ - arn:aws:iam::765374464689:role/magchen-operator-kube-system-capa-controller-manager
+ - arn:aws:iam::765374464689:role/magchen-operator-kube-system-control-plane-operator
+ - arn:aws:iam::765374464689:role/magchen-operator-kube-system-kms-provider
+ - arn:aws:iam::765374464689:role/magchen-operator-openshift-image-registry-installer-cloud-creden
+ - arn:aws:iam::765374464689:role/magchen-operator-openshift-ingress-operator-cloud-credentials
+ - arn:aws:iam::765374464689:role/magchen-operator-openshift-cluster-csi-drivers-ebs-cloud-credent
+ - arn:aws:iam::765374464689:role/magchen-operator-openshift-cloud-network-config-controller-cloud
+ - arn:aws:iam::765374464689:role/magchen-operator-kube-system-kube-controller-manager
+Managed Policies:           Yes
+State:                      waiting (Waiting for user action)
+Private:                    No
+Delete Protection:          Disabled
+Created:                    Oct  1 2024 09:48:52 EDT
+User Workload Monitoring:   Enabled
+OIDC Endpoint URL:          https://rh-oidc-dev.s3.us-east-1.amazonaws.com/2d01vc5khcasnmdq11a7m0d8uri93cjs (Managed)
+Audit Log Forwarding:       Disabled
+External Authentication:    Disabled
+Etcd Encryption:            Disabled
+Registry Configuration:
+ - Allowed Registries:      abc.com <1>
+ - Insecure Registries:     insecure.com <2>
+ - Allowed Registries for Import: <3>
+    - Domain Name:          quay.io
+    - Insecure:             true
+ - Platform Allowlist:      2e0auar98h9ajmd6ndlmlqr5lsibkm4v <4>
+    - Registries:           image-registry.openshift-image-registry.svc:5000,quay.io,registry.redhat.io
+----
+<1> `Allowed Registries`: Maggie Chen to provide description
+<2> `Insecure Registries`: Maggie Chen to provide description
+<3> `Allowed Registries for Import`: Maggie Chen to provide description
```

Comment:
"Limits the container image registries from which normal users can import images. The format should be a comma-separated list of 'domainName:insecure'. 'domainName' specifies a domain name for the registry. 'insecure' indicates whether the registry is secure or insecure."